xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-validation/xwiki-platform-model-validation-api/src/main/resources/ApplicationResources.properties

```diff
@@ -17,5 +17,9 @@
 # Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
 # 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 # ---------------------------------------------------------------------------
+entitynamevalidation.create.invalidnameError=Cannot create document {0} because its name does not respect the configured name strategy of the wiki.
+entitynamevalidation.create.invalidname.possiblesolution=Change the name or move the parent to another space.
+entitynamevalidation.create.invalidname.possiblesolutionForLink=Edit the link of current page so that it complies to the current name strategy.
+entitynamevalidation.create.invalidname.strategyInfo=The configured name strategy is: {0}. This strategy works as defined in the following: {1}
 entitynamevalidation.create.invalidname=Cannot create document {0} because its name does not respect the name strategy of the wiki. Change the name or move the parent to another space.
 entitynamevalidation.nametransformation.error=Error while transforming the page title to a correct page name: the exact page title will be used.
```

Comment:
So for me this should be: `entityNameValidation.nameTransformation.error`. xwiki.org is down but in the dev practices we have some rules defined.

Reply:
Note that this one is an old one so I can't change it.